allennlp/predictors/bidaf.py

```diff
@@ -38,3 +44,101 @@ def _json_to_instance(self, json_dict: JsonDict) -> Instance:
         question_text = json_dict["question"]
         passage_text = json_dict["passage"]
         return self._dataset_reader.text_to_instance(question_text, passage_text)
+
+    @overrides
+    def predictions_to_labeled_instances(self,
+                                         instance: Instance,
+                                         outputs: Dict[str, np.ndarray]) -> List[Instance]:
+        """
+        NAQANET has the following fields, answer_as_passage_spans,  answer_as_question_spans,
```

Comment:
This is explaining the implementation, not giving a user information about what the method does.  That means it should be a comment, not a docstring.  It should go down below where you say "For NAQANet".